server/src/organizations/organizations.controller.ts

```diff
@@ -43,11 +43,8 @@ export class OrganizationsController {
   }
 
   @Patch(':id')
-  async update(
-    @Param('id') id: string,
-    @Body() updateOrganizationDto: UpdateOrganizationDto,
-  ): Promise<Organization> {
-    return this.organizationsService.update(+id, updateOrganizationDto);
+  update(@Param('id') id: string, @Body() body: UpdateOrganizationDto): Promise<Organization> {
+    return this.organizationsService.update(+id, body);
```

Comment:
if your only changes in this file are a var name change then maybe it's better to just not touch it
also, i'd argue against this var name change as we seem to have this convention in the rest of your methods

Reply:
`git checkout origin/main -- server/src/organizations/organizations.controller.ts` will put it right back for ya

Reply:
Will do!